orderbook/src/database/orders.rs

```diff
@@ -201,6 +201,18 @@ struct OrdersQueryRow {
     signing_scheme: DbSigningScheme,
 }
 
+fn is_sell_order_filled(
```

Comment:
This is cool, but is there also an is buy order filled to accompany it?

Reply:
I can definitely add one, and I'll now that you mention it.
Just haven't because there's no special case for buy orders.